server/service.go

```diff
@@ -218,7 +228,7 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 	var wg sync.WaitGroup
 	for _, relay := range m.relays {
 		wg.Add(1)
-		go func(relayAddr string) {
+		go func(relayAddr string, relayPubKey types.PublicKey) {
```

Comment:
do we want to pass in the relay alltogether instead of the individual fields?

Reply:
This way, we can only read the data. It slows down a little bit the process due to copies but honestly not _that_ much.